Ideas-2025.md

```diff
@@ -312,6 +313,49 @@ Specifically, we would be working toward accomplishing the following:
 
 ------------
 
+# AI tools for reflection
+
+**Prerequisites**<br>
+ - Experience with Python
+ - Experience with LLMs/Chatbots
```

Comment:
If this would be in Music Blocks / Turtle Blocks, would some degree of proficiency in JS be necessary?

Reply:
Yes.